### PR TITLE
CORS configurations for score server allowing multiple origins at once

### PR DESCRIPTION
**Pull request by dahiyaAD**

### CODE REVIEW

File: score-server/src/main/resources/application.yml [hunk 1]
```diff
@@ -82,6 +82,12 @@ auth:
     # E.g. TIMEOUT(n) = TIMEOUT(n-1) * MULTIPLIER
     multiplier: 2.0
 
+management:
+  endpoints:
+    web:
+      cors:
+        allowedOrigins: http://example:8080, http://localhost:8081, http://hello:8080
```
Comment (leoraba) [inline score-server/src/main/resources/application.yml:89]: can we leave only localhost origin by default? I guess others are just an example, also would be nice to add a comment to describe the value could contain multiple origins separated by commas.

Comment (dahiyaAD) [reply score-server/src/main/resources/application.yml]: Done!

**Review by leoraba — APPROVED** [review]
👍

**Review by leoraba — APPROVED** [review]

**Review by joneubank — CHANGES_REQUESTED** [review]
Looking for feedback on my suggestion to reorganize the new properties added. Good idea? Bad idea?

**Comment by dahiyaAD** [comment]
Opened new PR : https://github.com/overture-stack/score/pull/369 since the job pr-merge stuck.

**Pull request closed without merging**